Notebooks/SentinelUtilities/SentinelUtils/crypto_utility.py

```diff
@@ -0,0 +1,37 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""
+cryto_utility module:
+This module provides encryption and decryption functionalities
+"""
+
+from cryptography.fernet import Fernet
+
+
+class CryptoUtility():
+    """ This class provides utility methods to encrypt and decrypt """
+    def __init__(self, key):
+        self.key = key
+
+    @staticmethod
+    def generate_key():
+        """ Generate key """
+
+        return Fernet.generate_key()
+
+    def encrypt_text(self, text):
```

Comment:
I think to avoid bad publicity it would be better to use terms like obfuscate, deobfuscate, obfuscation_seed. etc.